src/test/java/com/google/cloud/tools/gradle/appengine/core/CheckCloudSdkTaskTest.java

```diff
@@ -130,7 +141,8 @@ public void testCheckCloudSdkAction_versionFileException() {
   }
 
   @Test
-  public void testCheckCloudSdkAction_appEngineInstallationExceptions() {
+  public void testCheckCloudSdkAction_appEngineInstallationExceptions()
```

Comment:
Again, I probably could have named this something more specific.

Reply:
I think my auto-fix messed up here too.